### ISSUE TITLE
The package published to npm doesn't work because apparently it doesn't have the code (only publish LICENSE, README and package.json)

### ISSUE THREAD

**Issue by rigobauer** [semantic-release/semantic-release#615]
# Bug report

## Current behavior

I have a react component project (created with nwb). When I create the new release using semantic-release, the package is updated and published to npm *without the code* (just publish LICENSE, README and package.json). If I set a new version manually in package.json and run `npm publish`, it works without problems and I can install the package in another project.
I have the list of directories configured in package.json, but I've also tried adding a .npmignore file, without success. I've even recreate the whole project from scratch with nwb, but same results.

## Expected behavior

It should publish the package correctly, with all the code.

## Environment

- **semantic-release** version: 12.2.2
- CI environment: Travis
- Plugins used: 
- **semantic-release** configuration: https://github.com/rigobauer/react-abc2svg
- CI logs: https://travis-ci.org/rigobauer/react-abc2svg/builds/332199645

**Comment by pvdlg** [comment]
It seems you have a `build` script in your `package.json`. I guess you have to run it in the CI to produce the files you want to publish?

It seems this script is not run. You can run script in the [prePublish](https://docs.npmjs.com/misc/scripts#prepublish-and-prepare) hook, so it will always be run before publishing.

**Comment by travi** [comment]
also, in case you are running semantic-release from the `publish` step on travis, make sure you set `skip_cleanup: true` or travis will cleanup the files that you built before publishing.

**Comment by rigobauer** [comment]
OMG! You're totally right... I adding the 'npm run build' in the after_success solve it (and makes total sense).

Just as a desperate try to justify myself and doesn't look so dumb, I thought that Travis was doing that build automatically, because I have another component project made using the exactly same process, and that package was ok (and that's what was driving me crazy and make me focus in the wrong places).

After reading @pvdlg answer, I re-check everything and then realize that I made the first publish of that package manually, and after that I incorporate semantic-release. The thing is that the next commits weren't related to new features or fixes, so obviously semantic-release didn't generate a new release, and the original package was still there.

I had already tried the skip_cleanup parameter (thanks @travi), but without success. Obviously, without building the package there were nothing to skip!

Totally my fault! Sorry and thank you, guys!
